keyboards/gray_studio/bd60/info.json

```
@@ -0,0 +1,517 @@
+{
+    "keyboard_name": "BD60",
+    "url": "",
+    "maintainer": "edwardslau",
+    "manufacturer":"GrayStudio",
+    "processor": "STM32F072",
+    "diode_direction": "COL2ROW",
+    "bootloader": "stm32-dfu",
+       "usb": {
+        "vid":"0x4753",
+        "pid":"0x4060",
+        "device_version": "1.0.0"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "ws2812": {
+        "pin": "B10"
+    },
+    "matrix_pins": {
+        "cols": ["A3", "C14", "C13", "B9", "B8", "B7", "B6", "B5", "B4", "B3", "A15", "B0", "A8", "B15", "B14"],
+        "rows": ["B13", "B12", "A2", "A0", "A1"]
+    },
+    "rgblight": {
```

migrated from `config.h`
```suggestion
    "rgblight": {
        "layers": {
            "enabled": true
        },
```